tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
I don't agree with that. You can remove all methods apart from the constructor and it will still work. Conversely, if you use isEmpty, and it gets removed, these tests will break.

I cannot remember running into problems due to using `assertEquals`. I can remember running into problems due to tests using production code not under test.